study-builder/tenants/circadia/tenant.yaml

```diff
@@ -0,0 +1,50 @@
+tenant:
+  default_directory: Username-Password-Authentication
+
```

Comment:
Is the tenant created already? Can you make sure at the very least @MocanaAtBroad is added as an admin for the tenant?

Reply:
@yufengwng this tenant is already set up. I'll add Marco.